.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,45 @@
+
+name: 🐛 Bug report
+description: Create a report to help us improve our product
+title: '[BUG] <title>'
+labels: 'bug'
+
+
+body:
```

Comment:
Missing 2 fields
 - Describe the bug
 - Error

Reply:
You forgot do you think this will be fixed then two options saying no